### ISSUE TITLE
[Meta] Add Accessibility to the GitHub issue template for features

### ISSUE THREAD

**Issue by wtimme** [spacepandas/cineaste-ios#110]
**Is your feature request related to a problem? Please describe.**
The GitHub for creating new features helps me to clearly communicate my thoughts about a new feature. In my opinion, the template is lacking an "Accessibility* section.

Having such a section would help me to keep Accessibility in mind when writing a new issue.

**Describe the solution you'd like**
Modify the GitHub issue template for feature requests and add a section before "Additional context":
```markdown
**Accessibility**
Is there anything we need to keep in mind in terms of having an accessible UI?
(For example, do we need to specify `accessibilityLabel`s or `accessibilityValue`s?
```

**Describe alternatives you've considered**
We could also put this into the README as part of our "coding guidelines". For me, however, it helps to follow this template when creating new issues.

**Comment by fbernutz** [comment]
This is a good idea to keep accessibility in mind! 

But I think this makes more sense for a **Pull Request Template**, because there you have to check if you have implemented the needed accessibility features, like `accessibilityLabel`s or `accessibilityValue`s or tested dynamic type etc.

**Comment by fbernutz** [comment]
There is now a Pull Request Template with reminders to add and test accessibility features.
